File: app/src/main/java/com/ardakaplan/myapplication/ui/LifeCycleLogActivity.java
package com.ardakaplan.myapplication.ui;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;

import com.ardakaplan.myapplication.R;

/**
 * Created by Arda Kaplan at 4.08.2021 - 13:55
 * <p>
 * ardakaplan101@gmail.com
 */
public class LifeCycleLogActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.life_cycle_log_activity);
    }

    public static void open(Activity activity) {

        activity.startActivity(new Intent(activity, LifeCycleLogActivity.class));
    }

}

File: app/src/main/java/com/ardakaplan/testapp/rdalogger/ui/LifeCycleLogActivity.kt
package com.ardakaplan.testapp.rdalogger.ui

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import com.ardakaplan.testapp.rdalogger.R

/**
 * Created by Arda Kaplan on 21/02/2025 at 16:36.
 *
 * ardakaplan101@gmail.com
 */
class LifeCycleLogActivity : BaseActivity() {

    override fun onCreate(savedInstanceState : Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.life_cycle_log_activity)
    }

    companion object {

        fun open(activity : Activity) = activity.startActivity(Intent(activity, LifeCycleLogActivity::class.java))
    }
}